config/ember-try.js

```diff
@@ -6,27 +6,19 @@ const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 module.exports = async function () {
   return {
     scenarios: [
-      {
-        name: 'ember-lts-3.16',
-        npm: {
-          devDependencies: {
-            'ember-source': '~3.16.10',
-          },
-        },
-      },
       {
         name: 'ember-lts-3.20',
         npm: {
           devDependencies: {
-            'ember-source': '~3.20.7',
+            'ember-source': '~3.20.5',
           },
         },
       },
       {
         name: 'ember-lts-3.24',
         npm: {
           devDependencies: {
-            'ember-source': '~3.24.5',
+            'ember-source': '~3.24.3',
           },
         },
       },
```

Comment:
Recommend adding scenario for `ember-lts-3.28`.

Reply:
As project will rely on [ember-source 3.28](https://github.com/concordnow/ember-aria-tabs/pull/335/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R65) it's not useful to add an ember try scenario. `ember test` will already test 3.28